Sources/BraintreePaymentFlow/Public/BraintreePaymentFlow/BTPaymentFlowClient.h

```diff
@@ -2,7 +2,12 @@
 
 @class BTAPIClient;
 @class BTPaymentFlowRequest;
+
+#if __has_include(<Braintree/BraintreePaymentFlow.h>)
+#import <Braintree/BTPaymentFlowResult.h>
+#else
```

Comment:
This is _not_ a good practice generally but is being used temporarily to resolve the error: "attempting to use the forward class BTPaymentFlowResult as superclass of BTThreeDSecureResult" with `pod lib lint`.